rclcpp/src/rclcpp/duration.cpp

```diff
@@ -214,6 +214,13 @@ Duration::nanoseconds() const
   return rcl_duration_.nanoseconds;
 }
 
+Duration
+Duration::DURATION_MAX()
+{
+  Duration duration_max(std::numeric_limits<int32_t>::max(), 999999999);
```

Comment:
This seems like it would be easier to read as a one line function. No need to name the temporary variable, it could just be returned.